permission/core/cache.go

```diff
@@ -118,7 +132,7 @@ type OrgDetailInfo struct {
 }
 
 var syncStarted = false
-
+var emptyNodeId = enode.ID{}
```

Comment:
I don't like global variables, but it's just me.

Reply:
Agree, me too. Replaced to empty enode id value in if block. https://github.com/ConsenSys/quorum/pull/1578/commits/39fb51961fa51af64d0768e0b05c681a1a3b6d42#diff-a39234da6bbd13a5c02519dd2dff8aac2ea8a8d948ac58ed60c4ce5bd84f34b5R603